monai/apps/utils.py

```diff
@@ -37,6 +39,12 @@
 ]
 
 
+def _basename(p):
+    """get the last part of the path (removing the trailing slash if it exists)"""
+    sep = os.path.sep + (os.path.altsep or "") + "/ "
```

Comment:
Please double confirm whether it works fine for windows?
Thanks.

Reply:
sure, and the '/' is mainly for getting filename from the URLs